src/test/java/hw2/base/Const.java

```diff
@@ -0,0 +1,40 @@
+package hw2.base;
+
+public class Const {
+
+    //names or titles
+    public static final String titleExpected = "Home Page";
+    public static final String login = "Roman";
+    public static final String password = "Jdi1234";
+    public static final String userName = "ROMAN IOVLEV";
+    public static final String menuButtons = "HOME\n"
+            + "CONTACT FORM\n"
+            + "SERVICE\n"
+            + "METALS & COLORS";
+    public static final String menuButtonsOnTheLeft = "Home\n"
+            + "Contact form\n"
+            + "Service\n"
+            + "Metals & Colors\n"
+            + "Elements packs";
+
+    //locators
+    public static final String userIcon = "profile-photo";
+    public static final String userLogin = "name";
+    public static final String userPassword = "#password";
+    public static final String enterButton = "login-button";
```

Comment:
**Best practice** to use Enums for login users.

You must create enum "LoginUser" with fields `login, password, username`